src/lib/env.private.ts

```
@@ -5,11 +5,11 @@ import { env } from "$env/dynamic/private";
-        return value;
-    }
-}
\ No newline at end of file
+	static getOrThrow(key: keyof typeof env): string {
+		const value = env[key];
+		if (!value) {
+			throw new Error(`Missing environment variable: ${key}`);
+		}
+		return value;
+	}
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)